binscripts/rvm-installer

```diff
@@ -27,7 +27,9 @@ rvm_install_initialize()
 
 log()  { printf "%b\n" "$*"; }
 debug(){ [[ ${rvm_debug_flag:-0} -eq 0 ]] || printf "%b\n" "$*" >&2; }
+notice() { log "$*" >&2 ; }
```

Comment:
If we writing to STDERR, maybe we should call this function `error()`?

Reply:
I've added a `warn()`, let me know what you think.  I can rename to `error()` if you like.